src/Input/TextInputEXT.cs

```diff
@@ -37,11 +37,28 @@ public static class TextInputEXT
 
 		#region Public Static Methods
 
+		/// <summary>
+		/// Returns if text input state is active
+		///
+		/// Note: For on-screen keyboard, this may remain true on
+		/// some platforms if an external event closed the keyboard.
+		/// In this case, check IsScreenKeyboardShow instead.
+		/// </summary>
+		/// <returns>True if text input state is active</returns>
 		public static bool IsTextInputActive()
 		{
 			return FNAPlatform.IsTextInputActive();
 		}
 
+		public static bool IsScreenKeyboardShown(GameWindow window)
```

Comment:
Let's make this the direct `IntPtr`, so that the API doesn't depend directly on the Game classes.

Reply:
I was originally thinking that but GameWindow doesn't expose the IntPtr to the window. Not sure how this would be passed in from the developer's side?

Reply:
Game.Window.Handle will have it!

Reply:
Ah yes. Ok will update PR tomorrow.